index.html

```diff
@@ -594,52 +596,57 @@
 S_\mathrm{subsurface} = \mathrm{Slab}(f_\mathrm{dielectric}, V^\infty_\mathrm{subsurface}) \ .
 \end{equation}
 
-As for the [glossy-diffuse](index.html#model/dielectricbase/glossy-diffuse) slab and the [translucent-base](index.html#model/dielectricbase/translucentbase), the subsurface is bounded by a dielectric interface with BSDF $f_\mathrm{dielectric}$, which generates the primary specular lobe parametrized via the "specular" parameters as described in the Dielectric base section.
+As for the [glossy-diffuse](index.html#model/dielectricbase/glossy-diffuse) slab and the [translucent-base](index.html#model/dielectricbase/translucentbase), the subsurface is bounded by a dielectric interface with BSDF $f_\mathrm{dielectric}$, which generates the primary specular lobe parametrized via the "specular" parameters as described in the Dielectric base section. Superimposed on this is the reflection generated by scattering in the subsurface medium. In this case the subsurface medium $V^\infty_\mathrm{subsurface}$ is given a parametrization which is particularly convenient for controlling the volumetric effect of dense subsurface scattering:
```

Comment:
"As for" is a little ambiguous here because it could also be interpreted as "Regarding" (i.e. "Regarding the glossy–diffuse slab and the translucent-base, ....").

But in this case "As for" actually means "As is the case for" or "As in the case of" or "As is true for". So maybe we could tweak the wording to clarify.

Reply:
I wonder if there's a better word than "Superimposed" here. "Superimposed on this" seems to suggest that the subsurface scattering is on top of the specular reflection. A few alternatives: "Coupled with this" or "Combined with this" or "Alongside this".

Reply:
In https://github.com/AcademySoftwareFoundation/OpenPBR/pull/171/commits/ff04b4249f510c023c8a0573cec9f2dda032540f:

>But in this case "As for" actually means "As is the case for" or "As in the case of" or "As is true for". So maybe we could tweak the wording to clarify.

Tweaked to "As in the cases of"..

> A few alternatives: "Coupled with this" or "Combined with this" or "Alongside this".

I tried something a bit more colloquial (arguably a bit wordy, but clear I hope):

>Combined with this is the reflection generated by light which is transmitted through the dielectric interface into the underlying embedded subsurface medium, where it scatters around and eventually transmits back out.